packages/calcite-components/.stylelintrc.json

```diff
@@ -40,6 +40,30 @@
       {
         "ignoreTypes": ["/^calcite-/"]
       }
+    ],
+    "scss/at-function-pattern": [
+      "^([a-z][a-z0-9]*)(-[a-z0-9]+)*$",
+      {
+        "message": "Expected class selector to be kebab-cased."
```

Comment:
All messages say `Expected class selector`, which is not accurate. Can we simplify these to "<target> should be kebab-cased"? 

The message does appear next to the name, so we could also use `Name should be kebab-cased`. I'll defer to you on this.